client/src/app/+my-account/my-account-videos/my-account-videos.component.ts

```diff
@@ -81,7 +81,7 @@ export class MyAccountVideosComponent extends AbstractVideoList implements OnIni
 
   async deleteSelectedVideos () {
     const toDeleteVideosIds = Object.keys(this.checkedVideos)
-                                    .filter(k => this.checkedVideos[ k ] === true)
+                                    .filter((k: any) => this.checkedVideos[ k ] === true)
```

Comment:
string (it's not inferred?)

Reply:
idk, but I get an error without anything. But will change it to string if this is possible